### ISSUE TITLE
Fix Placeholder view helper return value

### ISSUE THREAD

**Issue by weierophinney** [laminas/laminas-view#7]
This PR aims to fix the bug, outlined in the following issue: https://github.com/zendframework/zend-view/issues/16

The changes will allow `$this` as a return value of the view helper, when no name is passed as an argument, thus providing access to the methods inside the view helper. For this reason, more tests have been added to `ZendTest\View\Helper\PlaceholderTest`.

---

Originally posted by @thexpand at https://github.com/zendframework/zend-view/pull/162

**Comment by weierophinney** [comment]
What could we do about the failing Travis build?
If we use `zend-mvc ^2.7` it's failing on PHP 7.2 with `DEPS=lowest`.
If we use `zend-mvc ^2.7.13` it's failing on lower PHP versions, because of the minimum stability.

Should I revert the changes, made to the `composer.json`?

**EDIT:**
I have reverted my changes. Now the `composer.json` is the same it was in the beginning. I saw that other PRs are also having this issue.

---

Originally posted by @thexpand at https://github.com/zendframework/zend-view/pull/162#issuecomment-413132906
